server/repository/repositories.go

```diff
@@ -0,0 +1,68 @@
+//TODO: connect with the DB
+package repository
+
+import (
+	"fmt"
+
+	"github.com/src-d/code-annotation/server/model"
+)
+
+var assignments []*model.Assignment
+
+var ErrNoAssignmentsInitialized = fmt.Errorf("No assignments initialized")
+
+func GetExperimentById(ID int) (*model.Experiment, error) {
+	return &model.Experiment{
+		ID,
+		fmt.Sprintf("Experiment#%d", ID),
+		fmt.Sprintf("Description about experiment#%d", ID),
+	}, nil
+}
+
+func GetAssignmentsFor(userID int, experimentID int) ([]*model.Assignment, error) {
+	if len(assignments) == 0 {
+		return []*model.Assignment{}, ErrNoAssignmentsInitialized
+	}
+
+	return assignments, nil
+}
+
+func CreateAssignmentsFor(userID int, experimentID int) ([]*model.Assignment, error) {
+	assignments = []*model.Assignment{
+		&model.Assignment{ID: 1, UserID: userID, PairID: 1, ExperimentID: experimentID},
+		&model.Assignment{ID: 2, UserID: userID, PairID: 2, ExperimentID: experimentID},
+		&model.Assignment{ID: 3, UserID: userID, PairID: 1, ExperimentID: experimentID},
+		&model.Assignment{ID: 4, UserID: userID, PairID: 2, ExperimentID: experimentID},
+		&model.Assignment{ID: 5, UserID: userID, PairID: 1, ExperimentID: experimentID},
+		&model.Assignment{ID: 6, UserID: userID, PairID: 2, ExperimentID: experimentID},
+		&model.Assignment{ID: 7, UserID: userID, PairID: 1, ExperimentID: experimentID},
+		&model.Assignment{ID: 8, UserID: userID, PairID: 2, ExperimentID: experimentID},
+	}
+
+	return assignments, nil
+}
+
+func UpdateAssignment(assignmentId int, assignment *model.Assignment) error {
```

Comment:
looks like you missed my comment before.
What behavior should I expect from this method if `assignmentId` != `assignment.ID`?
I don't have a correct answer.
That's why I suggested to change func signature.

Reply:
From the specs https://github.com/src-d/code-annotation/issues/3#issuecomment-357760945, the assignment request comes with no ID, so [it is not considered](https://github.com/src-d/code-annotation/pull/16/files/28c54c4e696585b8342c42af4e446eb5e7b8959a#diff-1731f4c3708fc9f3243464af602a7b16R58)
Same happens with `answer.PairID`: it is ignored.
It will be ensured in the tests too.

Anyhow, I'll rollback to 
```
func UpdateAssignment(assignmentId int, answer string, duration int) error
```
That is more clear and guarantee our requirements by design

Reply:
we need to add `id` to an assignment. Because FE needs to update it. RESTful way to do it is to call `PUT .../assignments/{id}`